redis/deprecated/redisbp/example_monitored_client_test.go

```diff
@@ -3,68 +3,49 @@ package redisbp_test
 import (
 	"context"
 
-	"github.com/go-redis/redis/v7"
+	"github.com/go-redis/redis/v8"
 	opentracing "github.com/opentracing/opentracing-go"
 	"github.com/reddit/baseplate.go/metricsbp"
 	"github.com/reddit/baseplate.go/redis/deprecated/redisbp"
 	"github.com/reddit/baseplate.go/tracing"
 )
 
-// Service is an example go-kit service to help demonstrate how to use
-// redisbp.type MonitoredCmdableFactory in a service.
```

Comment:
not sure why do we want to remove this comment, you can probably update it to be more accurate, but please don't remove it.